pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -173,6 +173,10 @@
     public static final int DEFAULT_MAX_REDUCE_THREADS_PER_QUERY =
         Math.max(1, Math.min(10, Runtime.getRuntime().availableProcessors() / 2)); // Same logic as CombineOperatorUtils
 
+    // used for SQL GROUP BY ORDER BY
+    public static final String CONFIG_OF_BROKER_SQL_GROUPBY_TRIM_THRESHOLD = "pinot.broker.sql.groupby.trim.threshold";
```

Comment:
Can we reuse existing PQL configs?

Reply:
@mayankshriv 
Broker reduce also needs to be optimized the same way as server combine since both use indexed table. The PQL config which this PR is reusing on the server for SQL GROUP BY is in ServerConf and there is no way for it to be available to Broker. Secondly, PQL never had a broker config for GROUP BY. Hence, I introduced a new config.

Reply:
(optional) Suggest removing the `sql` (rename to `pinot.broker.groupby.trim.threshold`) because we are going to deprecate pql and sql will be the standard